config/accounts.go

```diff
@@ -25,7 +25,7 @@ type Account struct {
 	ID                      string                               `mapstructure:"id" json:"id"`
 	Disabled                bool                                 `mapstructure:"disabled" json:"disabled"`
 	CacheTTL                DefaultTTLs                          `mapstructure:"cache_ttl" json:"cache_ttl"`
-	EventsEnabled           bool                                 `mapstructure:"events_enabled" json:"events_enabled"`
+	EventsEnabled           *bool                                `mapstructure:"events_enabled" json:"events_enabled"`
```

Comment:
Can we add an assertion for the default value? And also for the 3 possible values including `nil`?
```
145   func TestDefaults(t *testing.T) {
146       cfg, _ := newDefaultConfig(t)
147  
148       cmpInts(t, "port", cfg.Port, 8000)
149   *-- 31 lines: cmpInts(t, "admin_port", cfg.AdminPort, 6060)-------------------------------------------------------------
180  
181       cmpBools(t, "account_defaults.price_floors.enabled", cfg.AccountDefaults.PriceFloors.Enabled, false)
182       cmpInts(t, "account_defaults.price_floors.enforce_floors_rate", cfg.AccountDefaults.PriceFloors.EnforceFloorsRate, 100)
183       cmpBools(t, "account_defaults.price_floors.adjust_for_bid_adjustment", cfg.AccountDefaults.PriceFloors.AdjustForBidAdjustment, true)
184       cmpBools(t, "account_defaults.price_floors.enforce_deal_floors", cfg.AccountDefaults.PriceFloors.EnforceDealFloors, false)
185       cmpBools(t, "account_defaults.price_floors.use_dynamic_data", cfg.AccountDefaults.PriceFloors.UseDynamicData, false)
    +     cmpBools(t, "account_defaults.events_enabled", cfg.AccountDefaults.EventsEnabled, false)
186  
config/config_test.go
```